api/v1alpha1/service_fence.proto

```diff
@@ -83,6 +83,10 @@ message ServiceFenceSpec {
     map<string, RecyclingStrategy> host = 1;
     // Switch to render servicefence as sidecar
     bool enable = 2;
+    // services in these namespaces are all stable dependency
+    repeated string namespace = 3;
```

Comment:
A single `namespace` is easy to misunderstand as the ns which this sf will affect. What about `namespaces` or `namespaceSelector`?

Reply:
change to namespaceSelector